dipy/viz/ui.py

```diff
@@ -3033,6 +3050,24 @@ def _set_position(self, coords):
             (0, num_newlines * self.font_size * 0.5)
         offset = (self.button.size[0] + self.button_label_gap, 0)
         self.text.position = coords + offset
+    
+    def toggle(self, i_ren, obj, element):
+        if self.checked:
+            self.deselect()
+        else:
+            self.select()
+
+        i_ren.force_render()
+
```

Comment:
With radio buttons, this will not work.
- When an option that is already checked is clicked, we don't want anything to change.
- When an unchecked option gets clicked, all other options should be deselected.

Reply:
See most recent commit. I've adapted the code for `RadioButton._handle_option_change`.